cmd/yurt-tunnel-agent/app/start.go

```diff
@@ -91,6 +94,17 @@ func Run(cfg *config.CompletedConfig, stopCh <-chan struct{}) error {
 	}
 	agentCertMgr.Start()
 
+	// 2.1. waiting for the certificate is generated
+	_ = wait.PollUntil(15*time.Second, func() (bool, error) {
```

Comment:
Is 15 second poll interval too long?

Reply:
I did it to reduce qps pressure of api-server. I don't want to see an avalanche when thousands of node comes(or other situations). 5 second or 15 second will not hurt user-experience, but an avalanche will.

Reply:
Everybody waits 15 seconds does not eliminated the bursty quests but only delay them. You will need to add random delay to smooth the QPS or just give a reasonable delay for getting a signed certificate.

Reply:
let's set it to 5 second first
i will update it to a random delay sometime later, when I'm free..